_alerts/2015-07-04-spv-mining.html

```diff
@@ -9,7 +9,7 @@
 update: 2015-07-06 02:00.  All times are UTC.</em></p>
 
 **Note: this alert is on-going: the situation has not yet been
-resolved. ([Update #1](#update-1))**
+resolved. ([Read more](#cause))**
```

Comment:
Maybe this is better: http://dg0.dtrt.org/en/alert/2015-07-04-spv-mining#update-1

018724d3a5d001ac8cb4c721e558366fb6be5a10